data/mysql/schema.sql

```diff
@@ -625,7 +625,9 @@ CREATE TABLE `ezcontentobject` (
   KEY `ezcontentobject_lmask` (`language_mask`),
   KEY `ezcontentobject_owner` (`owner_id`),
   KEY `ezcontentobject_pub` (`published`),
-  KEY `ezcontentobject_status` (`status`)
+  KEY `ezcontentobject_status` (`status`),
+  KEY `ezcontentobject_section` (`section_id`),
+  KEY `ezcontentobject_id_section` (`id`, `section_id`)
```

Comment:
Possibly nitpick, but are you sure that you need this index ? And that you want to put the 'id' column first?

Reply:
ps: as a matter of fact, the index ('section_id', 'id') is used for all queries where the index ('section_id') would be used, so you would need 1 less index.
To be tested of course, but I think that most databases are able to figures this out

Reply:
> Possibly nitpick, but are you sure that you need this index ? And that you want to put the 'id' column first?

@gggeek TBH this is a little bit of InnoDB-dark-magic. it made a small difference in case of UPDATE-where-SELECT query that was used before. We tried to apply suggestions from [this doc](https://dev.mysql.com/doc/refman/8.0/en/innodb-deadlocks.html) to our case. However since I've separated these queries, I have no good point about keeping it.

I've updated PR - I've kept only `section_id` index as in general it might be useful.